### PR TITLE
[Lock] Add missing message in Exceptions

### PR DESCRIPTION
**Pull request by jderusse**
| Q             | A
| ------------- | ---
| Branch?       | maste
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Review by sstok — APPROVED** [review]
Travis failure is unrelated.

**Comment by sstok** [comment]
Status: reviewed

File: src/Symfony/Component/Lock/Lock.php [hunk 1]
```diff
@@ -75,7 +75,7 @@ public function acquire($blocking = false)
             return false;
         } catch (\Exception $e) {
             $this->logger->warning('Failed to lock the "{resource}".', array('resource' => $this->key, 'exception' => $e));
-            throw new LockAcquiringException('', 0, $e);
+            throw new LockAcquiringException('Failed to lock the resource.', 0, $e);
```
Comment (fabpot) [inline src/Symfony/Component/Lock/Lock.php]: What about `Failed to lock the %KEY% resource.`?

Comment (jderusse) [inline src/Symfony/Component/Lock/Lock.php]: Fixed,
For consistency, I use the format already defined in other log messages ` ... the "{resource}" lock`.
And I fixed some inconsistent logs message which didn't use the same vocabulary.

File: src/Symfony/Component/Lock/Lock.php [hunk 2]
```diff
@@ -58,24 +58,24 @@ public function acquire($blocking = false)
                 $this->store->waitAndSave($this->key);
             }
 
-            $this->logger->info('Lock successfully acquired for "{resource}".', array('resource' => $this->key));
+            $this->logger->info('Successfully acquire the "{resource}" lock.', array('resource' => $this->key));
```
Comment (fabpot) [inline src/Symfony/Component/Lock/Lock.php]: acquired?

**Review by xabbuh — APPROVED** [review]
👍

**Comment by fabpot** [comment]
Thank you @jderusse.

**Pull request merged**